tox.ini

```
@@ -1,7 +1,7 @@
 [tox]
 envlist =
-  py{39,310,311}-django{42,50}-postgres
-  py311-django{42,50}-mysql,
+  py{39,310,311}-django{42,51}-postgres
+  py311-django{42,51}-mysql,
```

`django50` debería seguir presente.

```suggestion
  py{39,310,311}-django{42,50,51}-postgres
  py311-django{42,50,51}-mysql,
```